src/renderer/components/atoms/button/types.ts

```diff
@@ -0,0 +1,34 @@
+import type {
+  ReactNode,
+  ButtonHTMLAttributes,
+  AnchorHTMLAttributes,
+} from 'react';
+
+import type { LinkProps } from 'react-router-dom';
+
+export type ButtonType = 'primary' | 'secondary' | 'warning';
+export type ButtonVariants = 'big' | 'small';
+
+export interface SimpleButtonProps
```

Comment:
I think this should be named `ButtonProps`, right?

Reply:
Oh yeah, fixed